pkg/network/server.go

```diff
@@ -321,8 +330,12 @@ func (s *Server) Shutdown() {
 		s.notaryRequestPool.StopSubscriptions()
 	}
 	close(s.quit)
+	<-s.broadcastTxFin
+	<-s.runProtoFin
 	<-s.relayFin
 
+	s.txHandlerLoopWG.Wait()
```

Comment:
Don't need a new line here, it's the same logic block (waiting for all dependent routines).